pkg/cmd/server/origin/controller.go

```diff
@@ -148,5 +215,40 @@ func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controll
 	}
 	ret["openshift.io/service-serving-cert"] = serviceServingCert.RunController
 
+	sdnController := controller.SDNControllerConfig{
+		NetworkConfig: c.Options.NetworkConfig,
+	}
+	ret["openshift.io/sdn"] = sdnController.RunController
+
+	// Overrides the upstream "resourcequota" controller
+	ret["resourcequota"] = controller.RunResourceQuotaManager
```

Comment:
Add a comment about this overriding a kube controller, but we have it here for now because it messes with openshift types.